lib/cosmos/tools/packet_viewer/packet_viewer.rb

```diff
@@ -259,6 +268,32 @@ def file_options
                                                 @polling_rate, 0, 1000, 1, nil)
     end
 
+    def edit_definition
+      # Grab all the cmd_tlm_files and processes them in reverse sort order
+      # because typically we'll have cmd.txt and tlm.txt and we want to process
+      # tlm.txt first
+      found = false
+      System.targets[@target_select.text].cmd_tlm_files.sort.reverse.each do |filename|
+        # Skip partials which begin with an underscore
+        next if File.basename(filename)[0] == '_'
+        file = File.read(filename)
+        if file.include?("TELEMETRY #{@target_select.text} #{@packet_select.text}")
```

Comment:
You should wildcard the target name in the config files because it is not used, and is often aliased.